### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by flakeman2** [neagle/gokibitz#63]
I forget to check gokibitz.com often to see if there are new comments.  It would be great if there was a way to be notified if/when there are new comments or game reviews.

Options to be notified:
 
1. Email once a day when there are new comments
2. Email for each new comment (maybe only email if the user is NOT on the site)
3. Add ability to 'follow' other users and be notified when they post a new game review.

I think this will help drive greater participation and keep pulling people back to the site and provide greater activity.  Thanks for the great site and keep up the good work!

Devin (flakeman2)

**Comment by neagle** [comment]
Yeah, I need to get this done. No question that it'd be a great feature. The reason it's taking a while is that I'm not really validating anyone's email, as it stands, and that I also need to build a settings page. But this is definitely a feature _I_ want, just as someone who uses the site. I'll try to make it happen!

**Comment by precipice** [comment]
+1, would love this. Thanks!
